code/modules/projectiles/gun.dm

```diff
@@ -239,6 +243,25 @@
 		user.update_inv_r_hand()
 	feedback_add_details("gun_fired","[src.type]")
 
+/obj/item/weapon/gun/proc/process_laser(atom/target, mob/living/user, params)
+	if(!pointer_icon)
+		return
+
+	var/turf/targloc = get_turf(target)
+
+	//laser pointer image
+	var/list/showto = list()
+	for(var/mob/M in range(7,targloc))
```

Comment:
How would you fix this? This was taken from laser pointer code I think and it's not easy to test for this anyway.